Content/1.1.3 - PreAuthorization Review/preAuthorizationReview.json

```diff
@@ -0,0 +1,20 @@
+{
+	"preAuthorizationReview_title": "Review your Pre-Authorization",
+	"preAuthorizationReview_subtitle": "Proposed by %@",
+	"preAuthorizationReview_loading": "Incoming Pre-Authorization",
+	"preAuthorizationReview_fees_title": "Pre-authorization will be returned to %@ for processing.",
+	"preAuthorizationReview_fees_subtitle": "Network fees will be paid by the dApp",
+	"preAuthorizationReview_expiration_atTime": "Valid for the next **%@**",
+	"preAuthorizationReview_expiration_afterDelay": "Valid for **%@ after approval**",
+	"preAuthorizationReview_expiration_expired": "Pre-authorization is no longer valid",
+	"preAuthorizationReview_timeFormat_day": "1 day",
+	"preAuthorizationReview_timeFormat_days": "%d days",
+	"preAuthorizationReview_timeFormat_hour": "%@ hour",
```

Comment:
Maybe %d?

Reply:
We need it to be `%@` since we are not gonna pass a digit, but a String indicating `HH:MM`, so it will look as `1:24 hour` or `10:52 hours`

Reply:
Oh ok, I'm not used to this notation, so I'm wondering if it's not misleading for the users also. `1 hour 24 minutes` or `1h24m` looks better in my opinion. What do you think?

Reply:
Personally I like it, and it is what Aftab and Matt have defined, see [thread](https://rdxworks.slack.com/archives/C031A0V1A1W/p1729519086728179?thread_ts=1729161378.302959&cid=C031A0V1A1W) for reference.

Reply:
Sure, I didn't know if it was still up for discussion or not. It's good that it's already been agreed upon. Thanks!